### PR TITLE
Update feedback-provide-microsoft.md

### PR DESCRIPTION
**Pull request by msbemba**
removed place holder  text. Cosmetic change.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6895

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/misc/feedback-provide-microsoft.md [hunk 1]
```diff
@@ -42,8 +42,7 @@ If you are using [Feedback](https://feedbackportal.microsoft.com/), you can prov
 
 :::image type="content" source="../../media/sending-feedback-community-portal.png" alt-text="Screenshot: Selection of Microsoft products you can send feedback on":::
 
-Selecting a forum allows you to see all feedback related to that product, and you can upvote or comment on feedback that matches yours.  If you do not see feedback that is similar to your own, you can submit new feedback through the portal itself.
-add screenshot.
+Selecting a forum allows you to see all feedback related to that product, and you can upvote or comment on feedback that matches yours.  If you do not see feedback that is similar to your own, you can submit new feedback t 987hrough the portal itself.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/misc/feedback-provide-microsoft.md]:
```suggestion
Selecting a forum allows you to see all feedback related to that product, and you can upvote or comment on feedback that matches yours.  If you do not see feedback that is similar to your own, you can submit new feedback through the portal itself.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd  Please merge this PR. Thanks

**Pull request merged**